cli/shell.go

```diff
@@ -103,17 +107,52 @@ func ShellCmd() *cobra.Command {
 			kubeConfig, err = api.StartAPIServer(clusterData)
 			if err != nil {
 				return errors.Wrap(err, "failed to create api server")
-
 			}
 			defer os.RemoveAll(kubeConfig)
 
-			shellExec := exec.Command("bash")
-			shellExec.Stdin = os.Stdin
-			shellExec.Stdout = os.Stdout
-			shellExec.Stderr = os.Stderr
-			shellExec.Env = []string{fmt.Sprintf("KUBECONFIG=%s", kubeConfig)}
-			_ = shellExec.Run() // add error checking
+			shellCmd := os.Getenv("SHELL")
+			if shellCmd == "" {
+				return errors.New("SHELL environment is required for shell command")
+			}
+
+			shellExec := exec.Command(shellCmd)
+			shellExec.Env = os.Environ()
+			fmt.Printf("Starting new shell with KUBECONFIG Ctl-D when done to end the shell and sbctl server\n")
```

Comment:
log.Println

Reply:
I'm mixed on this one, the instructions are to remind the user how to exit so they don't end up with sbctl running forever. You could find it in the log file, but I'm leaning toward leaving it like the log message as an informational message for the user.